modules/router-certificates.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/certificate-reference.adoc
+
+[id="router-certificates_{context}"]
+= Router certificates
+
+[discrete]
+== Purpose
+
+The Ingress Operator uses certificates for:
+
+* Securing access to metrics for Prometheus.
+* Securing access to Routes.
+
+[discrete]
+== File path
+
+To secure access to Ingress Operator and Ingress Controller metrics, the Ingress
+Operator uses service serving certificates. The Operator requests a certificate
+from the service-ca controller for its own metrics, and the service-ca
+controller puts the certificate in a secret named metrics-tls in the
```

Comment:
Several things should be in backticks in this sentence. And in the next paragraph too.